python/GafferSceneUI/_SceneViewInspector.py

```diff
@@ -797,15 +799,23 @@ def __init__( self, plugs, warning=None, **kw ) :
 
 			# Label to tell folks what they're editing.
 
-			labels = { self.__plugLabel( p ) for p in plugs }
+			script = plugs[0].ancestor( Gaffer.ScriptNode )
+			labels = { p.relativeName( script ) for p in plugs }
 			label = GafferUI.Label()
+
 			if len( labels ) == 1 :
-				label.setText( "<h4>{}</h4>".format( next( iter( labels ) ) ) )
+				message = next( iter( labels ) )
+				toolTip = ""
 			else :
-				label.setText( "<h4>{} plugs</h4>".format( len( labels ) ) )
-				label.setToolTip(
-					"\n".join( "- " + l for l in labels )
-				)
+				editScope = plugs[0].ancestor( Gaffer.EditScope )
+				if editScope is not None :
+					message ="{} ({} plugs)".format( editScope.relativeName( script ), len( plugs ) )
+				else :
+					message ="{} plugs".format( len( plugs ) )
+				toolTip = "\n".join( "- " + l for l in labels )
```

Comment:
There's no guarantee that all the plugs are in this EditScope is there? In the Auto/None mode, the plugs could be anywhere...

Reply:
early peaking John sees inadvertently in-progress work.

Reply:
As you correctly noticed, this was indeed flawed! [This version](https://github.com/GafferHQ/gaffer/pull/3883/commits/564f9f98d0c7cb2dc194ab029a6913dcb585739b) has the intended logic.